sycl/test-e2e/ClusterLaunch/enqueueLaunchCustom_check_event_deps.cpp

```diff
@@ -0,0 +1,73 @@
+// Checks whether or not event Dependencies are honored by
+// piExtEnqueueLaunchKernelCustom
+// REQUIRES: cuda
+// RUN: %{build} -Xsycl-target-backend --cuda-gpu-arch=sm_90 -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/accessor.hpp>
+#include <sycl/buffer.hpp>
+#include <sycl/ext/oneapi/experimental/cluster_group_prop.hpp>
+#include <sycl/queue.hpp>
+#include <sycl/usm.hpp>
+
+#include <vector>
+
+template <typename T> void dummy_kernel(T *input, int N, sycl::nd_item<1> it) {
+#if defined(__SYCL_CUDA_ARCH__) && (__SYCL_CUDA_ARCH__ >= 900)
+  auto id = it.get_global_linear_id();
+  uint32_t cluster_dim;
+  asm volatile("mov.u32 %0, %%cluster_nctaid.x;" : "=r"(cluster_dim));
+
+  if (id < N) {
+    input[id] += static_cast<T>(cluster_dim);
+  }
+#endif
+}
+
+int main() {
+
+  std::vector<int> a(4096, -20);
```

Comment:
These tests deviate from the codebase style convention of naming variables with Uppercase letters. I'd also prefer not to have single letter variable names.

Reply:
Fixed

Reply:
You've changed this single letter variable name, but naming convention in the new e2e tests still doesn't match the codebase LLVM convention